understanding/error-identification.html

```diff
@@ -68,7 +68,7 @@ <h2>意図</h2>
                
             </ul>
             <div class="note"><div role="heading" class="note-title marker" aria-level="3">注記</div>
-               <div><p> 利用者が高すぎる値、または低すぎる値を入力し、その値をページ上のコードが自動的に許容範囲内に変更した場合、達成基準によって要求されるよう、利用者の入力エラーはまだ変更したところに記載される必要がある。変更された値を人に伝えるエラーの説明などは、この達成基準 (エラーの特定) 及び<a href="error-suggestion.html" class="understanding">達成基準 3.3.3 (エラー修正の提案)</a> の両方を満たすことになる。</p>
+               <div><p>もし利用者が高すぎる値、又は低すぎる値を入力し、その値がページ上のコードによって許容範囲内に収まるように自動的に変更されたとしても、この達成基準で要求されるように、利用者に入力エラーを説明する必要があるだろう。そのような変更された値を人に伝えるエラーの説明は、この達成基準 (エラーの特定) と<a href="error-suggestion.html" class="understanding">達成基準 3.3.3 (エラー修正の提案)</a> の両方を満たすことになる。</p>
```

Comment:
「そのような変更された値を人に伝えるエラーの説明」は「そのような」がどこにかかるのかわかりにくく、「そのような値」と読めてしまいそうです (実際には「そのようなエラーの説明」)。
読点を入れると多少良くなったりしますかね?

```suggestion
               <div><p>もし利用者が高すぎる値、又は低すぎる値を入力し、その値がページ上のコードによって許容範囲内に収まるように自動的に変更されたとしても、この達成基準で要求されるように、利用者に入力エラーを説明する必要があるだろう。そのような、変更された値を人に伝えるエラーの説明は、この達成基準 (エラーの特定) と<a href="error-suggestion.html" class="understanding">達成基準 3.3.3 (エラー修正の提案)</a> の両方を満たすことになる。</p>
```